packages/ecommerce-app-base/package.json

```diff
@@ -1,9 +1,13 @@
 {
   "name": "@contentful/ecommerce-app-base",
   "version": "1.0.1",
-  "main": "lib/",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
   "author": "Contentful GmbH",
   "license": "MIT",
+  "files": [
+    "lib"
```

Comment:
I think it's safer to use globs here to capture the files we want to include and reduce the package size